Makefile

```diff
@@ -16,7 +16,7 @@ doc:
 	@${DUNE} build @doc
 
 install:
-	${OPAM} reinstall cooltt
+	${OPAM} reinstall --working-dir cooltt
```

Comment:
This makes it so it installs the version with local edits, and not just those that are committed to the repo.

Reply:
wow awesome! didn’t know how to do that

Reply:
This may be preferable to the current setup, not sure, but FYI my setup is that I test cooltt (and redtt) through the vim mode, and I have this in my vimrc: `let g:cooltt_path = $HOME . '/Documents/cooltt/_build/install/default/bin/cooltt'`

Reply:
gotcha gotcha, that also works. And yeah, I didn't know about this either until I did `make install` and the command line feedback included a comment that it wasn't including the changes from the working directory and it told me to add the option if that wasn't my intention, haha.

Reply:
Yeah! One downside of my approach is that I occasionally forget that the installed version is ancient and confuse myself on the command line. :sunglasses:

Reply:
I also prefer to be extra sure in the way that Carlo describes, but this change is excellent regardless. The default (perverse!) behavior of opam is to install the version at the last commit, not what is actually in the directory.